content/md/en/docs/tutorials/connect-other-chains/parachain.md

```diff
@@ -1,19 +1,69 @@
 ---
-title: Connect to a relay chain
+title: Connect a local parachain
 description:
 keywords:
+  - parachain
+  - testnet
+  - local
+  - collator
+  - chain specification
 ---
 
-In this part of the workshop, we'll connect a parachain to our relay chain and add additional nodes.
+In this tutorial, we'll connect a parachain to a local relay chain.
 
 ## Tutorial objectives
 
 By completing this tutorial, you will accomplish the following objectives:
 
-- Register a para ID
-- Initialize a parachain
-- Start and connect a collator
-- Connect additional parachain nodes
+- Register a para ID for your parachain on a relay chain
+- Start block production of a parachain on a relay chain
+
+## Before you begin
+
+Before you begin, verify the following:
+
+- You have competed the [start a local relay chain](/tutorials/connect-other-chains/local-testnet/) tutorial, and have a running local network with no les than two validators.
+
+## Matching versions are critical
+
+You **must** use the _exact_ software versions defined in the [relay chain tutorial](/tutorials/connect-other-chains/local-relay/#matching-versions-are-critical) for this tutorial to function!
+For example, if you are using [Polkadot `release-v0.9.24`](https://github.com/paritytechtree/release-v0.9.24), use the `polkadot-v0.9.24` version of the [parachain template](https://github.com/substrate-developer-hub/substrate-parachain-template/tree/polkadot-v0.9.24).
+
+## Building the parachain template
+
+We will use the [Substrate parachain template](https://github.com/substrate-developer-hub/substrate-parachain-template) to launch a parachain on your local relay chain.
+The parachain template is similar to the [node template](https://github.com/substrate-developer-hub/substrate-node-template) used in solo chain development, so it should be quite familiar.
+Later, we will use this parachain template as the starting point for developing our own parachains.
+
+<!-- TODO NEW CONTENT -->
+<!-- See the guide on [converting a solo chain to a parachain](/reference/how-to-guides/parachains/convert) for details on how the parachain template was created, and how to convert your chain's logic (not state migrations!) to a parachain.** -->
+
```

Comment:
I'd prefer you to use real steps.

Reply:
Not sure what you mean in the context of a new HTG ...?

Reply:
https://github.com/substrate-developer-hub/substrate-docs/issues/1147